smack-extensions/src/main/java/org/jivesoftware/smackx/geoloc/packet/GeoLocation.java

```diff
@@ -308,126 +335,255 @@ public static GeoLocation from(FormField formField) {
         private String tzo;
         private URI uri;
 
+        /**
+         * Sets accuracy of horizontal GPS error in meters.
+         * @param accuracy : accuracy accuracy in meters.
```

Comment:
Do not use the colon here. I doubt that it will render will. I would also prefer no period at the end.

Instead of
```@param accuracy : accuracy accuracy in meters.```
do
```@param accuracy the accuracy in meters```

When in doubt, look how the JRE is documented, .e.g http://hg.openjdk.java.net/jdk8/jdk8/jdk/file/687fd7c7986d/src/share/classes/java/util/HashMap.java#l484